content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -42,7 +42,7 @@ The `spec` of a static Pod cannot refer to other API objects
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,
+This page assumes you're using {{< glossary_tooltip term_id="containerd" >}} to run Pods,
```

Comment:
Is there any containerd assumption in the revised page?

Reply:
This is a mistake on my part. Talking and typing is never good.